src/main/java/dukeApp/parse/Parse.java

```diff
@@ -0,0 +1,55 @@
+package dukeApp.parse;
+import dukeApp.files.*;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Parse {
+    String statement;
+
+    public Parse(String statement) {
+        this.statement = statement;
+    }
+
+    public boolean decode(TaskList tasks) {
+        String taskArray[] = new String[2];
+
+        if (statement.equals("bye")) {
+            return true;
+        }
+        else {
+            if (statement.equals("list") && tasks.getSize() != 0) {
+                tasks.printList();
+            }
+            //if empty throw error
+            else if (statement.equals("list")) {
+                Ui ui = new Ui();
+                DukeException error = new DukeException();
+                System.out.println(error.empty());
+                System.out.println();
+            }
+
+            //any command that's not list
+            else {
+                //task array is each line from input file
+                taskArray = statement.split(" ");
+                if (taskArray.length == 1) {
+                    Ui ui = new Ui();
+                    DukeException error = new DukeException();
+                    System.out.println(error.errorMsg(taskArray[0]));
+                    System.out.println();
+                }
```

Comment:
Don't use Egyptian style brackets

Reply:
hi, could I clarify why Egyptian style brackets shouldn't be used?